src/frontend/Ast.ml

```diff
@@ -222,7 +222,7 @@ type 's block = {stmts: 's list; xloc: Middle.Location_span.t [@ignore]}
 and comment_type =
   | LineComment of string * Middle.Location_span.t
   | BlockComment of string list * Middle.Location_span.t
-  | Comma of Middle.Location.t
+  | Separator of Middle.Location.t
```

Comment:
Mind adding a note on this constructor about its purpose? Something like `(** Seperator is used to denote the location of items like commas and keywords which don't have location information stored in the AST, but are useful for placing comments in printing *)`

(assuming my interpretation is correct)